templates/components/itilobject/actors/main.html.twig

```diff
@@ -180,18 +181,18 @@
                <input class="form-check-input" type="checkbox" id="use_notification" name="_notifications_use_notification" />
                <label class="form-check-label" for="use_notification">{{ __('Email followup') }}</label>
             </div>
-            <div id="alert_notification_disable" class='alert alert-info d-flex align-items-center mb-4 d-none' role='alert'>
-               <i class='ti ti-info-circle fs-2x'></i>
-               <span class='ms-2'>
-                  {{ __('Notifications are disabled because:') }}
-                  <ul>
+             {% set alert_message %}
+                 <ul>
                      <li>{{ __('User does not have an email address.') }}</li>
                      <li>{{ __('User has disabled notifications from its preferences.') }}</li>
                      <li>{{ __('Notifications are disabled in this entity.') }}</li>
-                  </ul>
-                  {{ __('However, you can reactivate the notifications for this ticket.') }}
-               </span>
-            </div>
+                 </ul>
+                 {{ __('However, you can reactivate the notifications for this ticket.') }}
+             {% endset %}
+             {{ alerts.alert_info(title: __('Notifications are disabled because:'), messages: [alert_message], options: {
```

Comment:
`alert_message` contains HTML code that will be escaped.

Reply:
In my testing, that is not the case.

When you set `alert_message` using `{%set %}` the twig variables inside are already automatically escaped and the HTML content is assumed to be intentional since it was directly written. Twig tries to avoid auto-escaping this content again and so it works as if the alert macro outputs the message with a raw filter in this case.

Reply:
My bad, I was not aware of this behaviour. I will redo my review then.